brewpi-script/scriptlibs/pinList.py

```diff
@@ -41,26 +41,47 @@ def getPinList(boardType, shieldType):
                    {'val': 20, 'text': 'A2', 'type': 'free'},
                    {'val': 21, 'text': 'A3', 'type': 'free'}]
     elif boardType == "uno" and shieldType == "revC":
-        pinList = [{'val': 6, 'text': ' 6 (Act 1)', 'type': 'act'},
-                   {'val': 5, 'text': ' 5 (Act 2)', 'type': 'act'},
+        pinList = [{'val': 0, 'text': ' 0', 'type': 'serial'},
+                   {'val': 1, 'text': ' 1', 'type': 'serial'},
                    {'val': 2, 'text': ' 2 (Act 3)', 'type': 'act'},
-                   {'val': 19, 'text': 'A5 (Act 4)', 'type': 'act'},
-                   {'val': 4, 'text': ' 4 (Door)', 'type': 'door'},
-                   {'val': 18, 'text': 'A4 (OneWire)', 'type': 'onewire'},
                    {'val': 3, 'text': ' 3', 'type': 'beep'},
+                   {'val': 4, 'text': ' 4 (Door)', 'type': 'door'},
+                   {'val': 5, 'text': ' 5 (Act 2)', 'type': 'act'},
+                   {'val': 6, 'text': ' 6 (Act 1)', 'type': 'act'},
                    {'val': 7, 'text': ' 7', 'type': 'rotary'},
                    {'val': 8, 'text': ' 8', 'type': 'rotary'},
                    {'val': 9, 'text': ' 9', 'type': 'rotary'},
                    {'val': 10, ' text': '10', 'type': 'spi'},
                    {'val': 11, ' text': '11', 'type': 'spi'},
                    {'val': 12, ' text': '12', 'type': 'spi'},
                    {'val': 13, ' text': '13', 'type': 'spi'},
-                   {'val': 0, 'text': ' 0', 'type': 'serial'},
-                   {'val': 1, 'text': ' 1', 'type': 'serial'},
                    {'val': 14, 'text': 'A0', 'type': 'free'},
                    {'val': 15, 'text': 'A1', 'type': 'free'},
                    {'val': 16, 'text': 'A2', 'type': 'free'},
-                   {'val': 17, 'text': 'A3', 'type': 'free'}]
+                   {'val': 17, 'text': 'A3', 'type': 'free'},
+                   {'val': 18, 'text': 'A4 (OneWire)', 'type': 'onewire'},
+                   {'val': 19, 'text': 'A5 (Act 4)', 'type': 'act'}]
+    elif boardType == "uno" and shieldType == "I2C":
```

Comment:
I'm assuming you stuck with shield type "I2C"? If so, I'll get this merged in.

Reply:
I did.